cra-rxjs-styled-components/src/components/dropdown/Dropdown.tsx

```diff
@@ -17,38 +15,20 @@ import {
   IconContainer,
 } from './Dropdown.styles';
 
-type AvatarImage = {
-  image: string;
-  alt?: string;
-};
 export default function Dropdown() {
   const signOutHandler = useSignOut();
-  const [avatar, setAvatar] = useState<AvatarImage>({ image: '' });
+  const user = useUser();
   const [openDropdown, setOpenDropdown] = useState(false);
   const toggleDropdown = () => setOpenDropdown(!openDropdown);
 
-  const request = (url: string) =>
-    fromFetchWithAuth(url, {
-      selector: (response: Response) => {
-        return response.json();
-      },
-    });
-
-  useEffect(() => {
-    request(`${GITHUB_URL_BASE}/user`)
-      .pipe(
-        tap((val) => {
-          setAvatar({ image: val.avatar_url, alt: val.name });
-        })
-      )
-      .subscribe();
-  }, []);
   return (
     <DropdownNav>
       <DropdownBtn onClick={toggleDropdown}>
         <UserDropdownAvatarContainer>
           <UserDropdownAvatarSpan>
-            <UserDropdownAvatarImage alt={avatar.alt} src={avatar.image} />
+            {user?.avatar_url && user.name && (
```

Comment:
We should add the optional chaining for the name prop as well as this might throw an exception 
`user?.name`

Reply:
For the first `?` tbh I added it to make typescript happy but about the user name the statement already checks on it if it's exist or not